scarlet/source.py

```diff
@@ -119,24 +121,28 @@ def __init__(
             Multiple of the backround RMS used as a
             flux cutoff for morphology initialization.
         """
+        if spectrum is None:
+            spectrum = get_pixel_sed(sky_coord, observations)
```

Comment:
This is the same problem again, I think. These are concatenated spectra if more than one observation is present

Reply:
Not relevant anymore.